### DISCUSSION TITLE
TY-01 Door Sensor - unable to flash (new MCU?)

### DISCUSSION THREAD

**Discussion by fraintt** [arendst/Tasmota#11455] [Q&A] [answered]
Hi,
I would like to ask a question about the TY-01 Door/Window Sensor, which I recently bought from Aliexpress. I tried to follow the Instructions provided by blakadder but failed in every attempt. I also noticed that the MCU is completely different than the one in the images from the template. I also identified the tx,rx,3v3,gnd an rst ports at the bottom by measuring with an multimeter and shorted the gnd and rst (they are different than in the template) but I was not able to flash it. I also tried to connect 3v3 to ch_en, didnt work, I also tried to de-solder the tx and rx pins of the mcu from the board completely, (not on the image, did it afterwards) but it also did not help... Im flashing with an arduino, and flashed some TYWE3S modules before without issues, so I do not think it is "not enough power" thing.. Any advice would be really helpful, thank you in advance. Attached is also the image of the door sensor with my findings. 

(This is my last attempt before I will try to de-solder the whole tywe3s module, I have 5 of these things, so 5 attempts? :D )

![IMG_0387](https://user-images.githubusercontent.com/33605993/112311061-8ee81900-8ca5-11eb-80d4-7a88e6d7ab8c.JPG)

**Comment by fraintt** [comment, marked as answer]
Nobody any suggestion?

**Reply by Jason2866** [reply]
You can try https://youtu.be/JE3vWB-rhFM

**Reply by ortegafernando** [reply]
> Nobody any suggestion?

probably this is yours: https://templates.blakadder.com/D06.html

**Reply by fraintt** [reply]
Hi Guys, thank you for the ideas. @ortegafernando yours worked, I do not know why I did not check this template.. Altought it is not the same as in the picture and also the board does not look exactly the same, it worked when desoldergin the one resistor. (Why then it didnt work when I de-soldered the mcu tx&rx pins directyl? remains unanswered) I flashed two of them sucessfully. Now to find the correct template and rules for the door opening, but I will figure this out.

Thank you again, it helped a lot!
